src/build/supportedExtensions.js

```diff
@@ -5,16 +5,16 @@ exports.extensions = {
     { icon: 'actionscript', extensions: ['as'] },
     { icon: 'ai', extensions: ['ai'], svg: true },
     { icon: 'angular', extensions: ['angular-cli.json'], filename: true, svg: true },
-    { icon: 'ng2_component_ts', extensions: ['.component.ts'], svg: true },
-    { icon: 'ng2_component_js', extensions: ['.component.js'], svg: true },
-    { icon: 'ng2_directive_ts', extensions: ['.directive.ts'], svg: true },
-    { icon: 'ng2_directive_js', extensions: ['.directive.js'], svg: true },
-    { icon: 'ng2_pipe_ts', extensions: ['.pipe.ts'], svg: true },
-    { icon: 'ng2_pipe_js', extensions: ['.pipe.js'], svg: true },
-    { icon: 'ng2_service_ts', extensions: ['.service.ts'], svg: true },
-    { icon: 'ng2_service_js', extensions: ['.service.js'], svg: true },
-    { icon: 'ng2_module_ts', extensions: ['.module.ts'], svg: true },
-    { icon: 'ng2_module_js', extensions: ['.module.js'], svg: true },
+    { icon: 'ng2_component_ts', extensions: ['.component.ts'], filename: true, svg: true },
```

Comment:
Seems you are correct. The test was complaining... and head was buzzing. Will revise.